CODEOWNERS

```diff
@@ -1 +1 @@
-@oss-integrations
+* @oss-integrations
```

Comment:
look like @oss-integrations does not have access

Reply:
added the prefix `astronomer/` which was needed for the team